Examples/Python/python/acts/examples/reconstruction.py

```diff
@@ -645,7 +639,7 @@ def addCKFTracks(
     outputDirCsv: Optional[Union[Path, str]] = None,
     outputDirRoot: Optional[Union[Path, str]] = None,
     selectedParticles: str = "truth_seeds_selected",
-) -> acts.examples.Sequencer:
+) -> None:
```

Comment:
I think this indicates that the function returns `None`, not that it doesn't return a value. If it really doesn't return a value, then I think this should be `NoReturn`, but I wonder if just not specifying a return type is OK - and clearer.

Reply:
Thanks for the correction. Sorry for the incorrect comment: I misread the stackoverflow.